requirements.txt

```diff
@@ -33,8 +33,10 @@ Pillow==6.1.0 ; python_version <= '3.7' and sys_platform == 'win32'
 Pillow==8.1.1 ; python_version > '3.7'
 polib==1.1.0
 psutil==5.6.6
+psutil==5.6.7 ; Windows 10 and python_version >= '3.9'
```

Comment:
That kind of environment marker will not work
https://www.python.org/dev/peps/pep-0508/#environment-markers